packages/compass-indexes/src/components/indexes-table/property-field.tsx

```diff
@@ -110,11 +112,11 @@ const PropertyField: React.FunctionComponent<PropertyFieldProps> = ({
         />
       )}
       {extra.status === 'inprogress' && (
-        <Badge variant={BadgeVariant.Blue}>In Progress...</Badge>
+        <Badge variant={BadgeVariant.Blue}>In Progress ...</Badge>
       )}
       {extra.status === 'failed' && (
         <ErrorBadgeWithTooltip
-          tooltip={String(extra.error)}
+          tooltip={extra.error ? String(extra.error) : ''}
```

Comment:
drive by, if error is `undefined` this actually would show a tooltip with the `undefined` text in it